### PR TITLE
RHINENG-12213 Update CPU and Memory Param names

### PR DESCRIPTION
**Pull request by saltgen**
We need to update CPU and Memory params in the Clowdapp files.

This is to ensure we utilize resources on ephemeral environments optimally.

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here. If the change depends on Kruize add details about that as well!

## Documentation update? :memo:

- [ ] Yes
- [x] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Does this change depend on specific version of Kruize
  - If yes what is the version no:
  - [ ] Is that image available in production or needs deployment?
- [ ] Bugfix
- [ ] New Feature
- [x] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Please check JIRA for additional details.

### CODE REVIEW

**Comment by saltgen** [comment]
For some reason the sources pod's readiness fails on the PR Check ~ Jenkins, changes look fine on EE though.

image

**Comment by saltgen** [comment]
The PR Check is failing, since memory max limit to request ratio per Pod is 2, but provided ratio is 4.000000, for a dependency `sources-api`.

File: scripts/samples/cost-mgmt/cee-enablement-demo.tar.gz
Comment (r14chandra) [inline scripts/samples/cost-mgmt/cee-enablement-demo.tar.gz:1]: @saltgen Is this file intentionally added?

Comment (saltgen) [reply scripts/samples/cost-mgmt/cee-enablement-demo.tar.gz:1]: Oops, I have removed the same, and the pr_check changes

**Review by r14chandra — APPROVED** [review]

**Pull request merged**